test/runtests.jl

```diff
@@ -1,9 +1,16 @@
 module TestRematch
 
+if VERSION >= v"0.7.0-DEV"
+    using InteractiveUtils
+else
+    AbstractRange = Range
+end
```

Comment:
You could also use `Compat.InteractiveUtils` for this, i think, which might be simpler! :)

```julia
julia> VERSION
v"0.6.4"

julia> Compat.InteractiveUtils
Compat.InteractiveUtils
```